Header header
string type_flag
float64 coord1
float64 coord2
float64 coord3
